<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GvQCUDASourceEditor</class>
 <widget class="QWidget" name="GvQCUDASourceEditor">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUDA - Program Source Editor</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTextEdit" name="_textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="_applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_compileButton">
       <property name="text">
        <string>Compile</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>201</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
